origin-bridge/util/attestations.py

```diff
@@ -7,12 +7,13 @@
 # http://web3py.readthedocs.io/en/stable/web3.eth.account.html?highlight=private#not-acceptable-for-production
 w3.eth.enable_unaudited_features()
 
+DONT_HASH_DATA_TOPICS = [4, 5]
```

Comment:
Why do we need to special case these ? It complicates the code...
What would be wrong with hashing the IPFS hash we get for Twitter/Airbnb ?

Reply:
If we hash the data field then it becomes unreadable to the DApp, and it won't be able to fetch the IPFS hash from that field. There is some additional complexity here and I will add explanation as a comment:
```
- IPFS hash is a base58 encoded string
- We store IPFS hashes in solidity claims in byte32 hex encoding to minimise gas cost. 
- byte32 is not string serialisable so it can not be transmitted in that form from discovery to the DApp
- discovery needs to transform ipfs hash to base32 hex encoding (that is how it is going to be stored in the contract) before signing the claim, and then send IPFS hash to the DApp in base58 encoding.
- this way claim has a correct signature if IPFS hash has byte32 hex encoding
- the DApp takes signature and other claim info and transforms the base58 encoded IPFS hash to base32 hex before submitting the claim to web3. 
```

Reply:
Yes, thanks for the details, it makes sense.